### PR TITLE
emit useful metrics from LSO

### PR DESCRIPTION
**Pull request by sp98**
metrics to be exported:

- [x]  When user edits the LocalVolume or LocalVolumset objects and one or more devices/PVs as a result are "orphaned", we should emit them as a metric.
- [ ]  When device name changes, we should emit such device name changes as a aggregate metrics.
- [x]  Total number of provisioned devices. (done for localvolumeset)
- [x]  Total number of discovered devices.
- [x]  Total number of unmatched devices.

Signed-off-by: Santosh Pillai <sapillai@redhat.com>

### CODE REVIEW

**Comment by gnufied** [comment]
@sp98 can you drop WIP from this PR?

File: deploy/monitoring/rbac.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: local-storage-metrics
+subjects:
+- kind: ServiceAccount
+  name: prometheus-k8s
+  namespace: openshift-monitoring
+---
```
Comment (gnufied) [inline deploy/monitoring/rbac.yaml:57]: Don't we need these rbac rules in CSV as well?

**Comment by gnufied** [comment]
Can you rebase this PR please?

**Comment by sp98** [comment]
Closing the PR. The changes are covered in PR 249

**Pull request closed without merging**